File: C# Matrix test/Calculator.cs
using System;

class Calculator
{
    public double dist( Vector2D vec1, Vector2D vec2 )
    {
        double diff = Math.Sqrt(Math.Pow(vec1.X - vec2.X, 2) + Math.Pow(vec1.Y - vec2.Y, 2));
        return diff;
    }
}

File: Matrix Test/Calculator.cpp
#include <math.h>
#include "Calculator.h"

Calculator::Calculator()
{
}

Calculator::~Calculator()
{
}

double Calculator::dist(Vector2D vec1, Vector2D vec2)
{
	//2点の距離を求める(三平方の定理)= √(x1-x2)²+ (y1-y2)²
	//斜辺(距離)
	double diff = sqrt(pow(vec1.x - vec2.x, 2) + pow(vec1.y - vec2.y, 2));
	return diff;
}
